api/v1alpha1/reaper_types.go

```diff
@@ -121,6 +121,12 @@ type ReaperSpec struct {
 
 	// PodSecurityContext applied to Reaper pods
 	PodSecurityContext *corev1.PodSecurityContext `json:"podSecurityContext,omitempty"`
+
+	// Volumes applied to the Reaper pods
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
We need to add a volume and mount for `/etc/reaper` but we don't need to change the CRD to do that. We can use an emptyDir volume for this. You need to add the volume mount [here](https://github.com/k8ssandra/reaper-operator/blob/master/pkg/reconcile/reconcilers.go#L520) and the volume [here](https://github.com/k8ssandra/reaper-operator/blob/master/pkg/reconcile/reconcilers.go#L545).

In a second iteration, we could make the volume and volume mount configurable in some way. We could allow the user to specify a ConfigMap for example and then have the operator add the volume and mount point for it.

Reply:
Removed vols and mounts from CRD and now baked into the deployment.